Mathlib/Combinatorics/SimpleGraph/Matching.lean

```diff
@@ -309,4 +317,79 @@ lemma exists_maximal_isMatchingFree [Finite V] (h : G.IsMatchingFree) :
   obtain ⟨Gmax, hGmax⟩ := Finite.exists_le_maximal h
   exact ⟨Gmax, ⟨hGmax.1, ⟨hGmax.2.prop, fun _ h' ↦ hGmax.2.not_prop_of_gt h'⟩⟩⟩
 
+/-- A graph `G` consists of a set of cycles, if each vertex is either isolated or connected to
+exactly two vertices. This is used to create new matchings by taking the `symmDiff` with cycles.
+The definition of `symmDiff` that makes sense is the one for `SimpleGraph`. This is why this
```

Comment:
when you say "is the one for `SimpleGraph`", I don't know what the other option being discarded is. Could you clarify?

Reply:
Clarified in the comment:)